src/app/features/card/card.js

```diff
@@ -69,14 +95,39 @@ export function FeedCard(props) {
             onMouseLeave={() => dispatch(mouseOutside(props.id))}
           >
             {doBlur ? (
-              <Typography
-                gutterBottom
-                className={classes.font}
-                variant="h5"
-                component="h5"
-              >
-                {crntTitle}
-              </Typography>
+              <div>
+                <Typography
+                  gutterBottom
+                  className={classes.font}
+                  variant="h5"
+                  component="h5"
+                >
+                  {crntTitle}
+                </Typography>
+                <List
+                  className={classes.myList}
+                  dense={true}
+                  disablePadding
+                  subheader={
+                    <ListSubheader
+                      component="div"
+                      id="nested-list-subheader"
+                      className={classes.listHeader}
+                    >
+                      Chapters
+                    </ListSubheader>
+                  }
+                >
+                  {Object.keys(chapters)
```

Comment:
Location of interest for #11

Reply:
The sorting done by the API is clearly working (to some extent) because the top chapter displayed is in fact the most recent. For Nagatoro anyways. Need to look over api calls to make sure that they're actually in a correctly descending order.